src/components/SudokuGrid/Row.js

```diff
@@ -0,0 +1,29 @@
+import Cell from "./Cell";
+import { v4 as uuidv4 } from "uuid";
+
+export default function Row({
+  currentSudokuNumbers,
+  rowNumber,
+  mask,
+  onCellInRowClick,
+}) {
+  return (
+    <div className="Sudoku__row">
+      {currentSudokuNumbers[rowNumber].map((number, columnNumber) => {
+        let editable = true;
+        if (mask[rowNumber][columnNumber] !== 0) {
+          editable = false;
+        }
```

Comment:
You could also do this
```suggestion
        const editable = mask[rowNumber][columnNumber] !== 0 ? false : true
```

And I would rename `editable` to `isEditable` <- the `is` indicates that the variable may hold a `boolean`